src/lib/process/api.ts

```diff
@@ -93,6 +93,6 @@ export const getProcess = async (
     id: processId,
     version: version,
     status: data.status,
-    restrictions: data.restrictions,
+    restrictions: data.program,
```

Comment:
we should look to rename `restrictions` to program where appropriate (not just here). Basically if we have an array of things that contain operations and restrictions it should change

Reply:
this has been done, was not 100% sure what we would like to use it, should have asked.